types/tipset_test.go

```diff
@@ -153,7 +154,12 @@ func TestTipSet(t *testing.T) {
 		// String shouldn't really need testing, but some existing code uses the string as a
 		// datastore key and depends on the format exactly.
 		assert.Equal(t, "{ "+b1.Cid().String()+" }", RequireNewTipSet(t, b1).String())
-		assert.Equal(t, "{ "+b1.Cid().String()+" "+b2.Cid().String()+" "+b3.Cid().String()+" }",
+
+		// DONOTMETGE the below assertion needed to be fixed after adding a timestamp to block.
```

Comment:
@anorth could you confirm my suspicion here, blame shows you as the author. Does the order of these matter?

Reply:
Yes, order matters (from SortedCidSet)

Reply:
You can robustify the test by actually using a sortedcidset to order them.

Reply:
This is fixed in #2908